exporters/stdout/exporter.go

```diff
@@ -0,0 +1,94 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stdout
+
+import (
+	"go.opentelemetry.io/otel/api/global"
+	apitrace "go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/export/trace"
+	"go.opentelemetry.io/otel/sdk/metric/controller/push"
+	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+type Exporter struct {
+	traceExporter
+	metricExporter
+}
+
+var (
+	_ metric.Exporter   = &Exporter{}
+	_ trace.SpanSyncer  = &Exporter{}
+	_ trace.SpanBatcher = &Exporter{}
+)
+
+// NewExporter creates an Exporter with the passed options.
+func NewExporter(options ...Option) (*Exporter, error) {
+	config, err := Configure(options...)
+	if err != nil {
+		return nil, err
+	}
+	return &Exporter{
```

Comment:
should there not be the option to disable the trace or metric exporter e.g. to send all metrics to stdout but not to send traces?

Reply:
I could imagine this, but I wonder if it's going to be used much. I can't imagine many uses for half of the stdout exporter. If you're setting up one and not the other, you could avoid calling InstallNewPipeline instead and call only one of the global.Set methods.

Reply:
I can imagine quite a few -- for one, in Honeycomb's case we might have a trace exporter we're fully confident in with no need to print values locally, but be testing our metric export pipeline and want to compare the values side by side with what stdout is returning. But we wouldn't want every trace to get printed to stdout.

or vice versa in the case of someone who is using metrics successfully but wants to test their trace export pipeline.

Reply:
If you see a use @lizthegrey I'll add it here. I thought that kind of enablement/disablement was going to be handled as part of the Views API, but can add something here for the time being.

Reply:
@lizthegrey PTAL